watir-rspec.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.version       = Watir::RSpec::VERSION
 
-  gem.add_dependency "rspec", "~>2.0"
+  gem.add_dependency "rspec", "~>3.0"
```

Comment:
Now, to think about it - do we really need it to increase to `3.0`? Do we have currently anything `RSpec 3` specific, which is not working in `RSpec 2`? Isn't `expect` syntax also working with `RSpec 2`? Not sure and i don't have anything against increasing the version number here, but just made me think that do we actually need to do that.